zeroex/asset_data_decoder_test.go

```diff
@@ -77,3 +77,20 @@ func TestDecodeERC1155AssetData(t *testing.T) {
 	}
 	assert.Equal(t, expectedDecodedAssetData, actualDecodedAssetData, "ERC1155 Asset Data properly decoded")
 }
+
+func TestDecodeERC20BridgeAssetData(t *testing.T) {
+	assetData := common.Hex2Bytes("dc1600f30000000000000000000000006b175474e89094c44da98b954eedeac495271d0f00000000000000000000000077c31eba23043b9a72d13470f3a3a311344d743800000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000000")
+
+	d := NewAssetDataDecoder()
+
+	var actualDecodedAssetData ERC20BridgeAssetData
+	err := d.Decode(assetData, &actualDecodedAssetData)
+	require.NoError(t, err)
+
+	expectedDecodedAssetData := ERC20BridgeAssetData{
+		TokenAddress:  common.HexToAddress("0x6b175474e89094c44da98b954eedeac495271d0f"),
+		BridgeAddress: common.HexToAddress("0x77c31eba23043b9a72d13470f3a3a311344d7438"),
+		Calldata:      common.Hex2Bytes("0x"),
```

Comment:
We should test that `calldata` is decoded correctly. From the source code (https://github.com/0xProject/0x-monorepo/blob/development/contracts/asset-proxy/contracts/src/bridges/ChaiBridge.sol#L39), we can see that any `calldata` that is provided is valid. It's entirely possible that users will come up with novel uses of calldata, so I think that we need to test that we can decode it.

Reply:
@jalextowle, it looks like the calldata is not used by the Chai Bridge Contract. How are we supposed to know how to decode it? My understanding is it's simply a byte-array that can include _any_ data. Is that incorrect?

Reply:
That’s correct. When I say “we should test that calldata is decoded correctly” I mean that we should decode it as bytes. The only change that is required is to add a new test where the bytes field isn’t empty.

Reply:
Added a test for it.